src/components/footer.js

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { Link } from "gatsby"
+import "./footer.css"
+import promiseFundLogo from "../images/promise-fund-logo.svg"
+
+export default function footer() {
+  return (
+    <footer className="footer">
+      <section className="footer-container">
```

Comment:
I'm not sure I would use `<section>` and `<article>` here, they don't really match the content inside. I think just normal divs are fine.